manage.py

```diff
@@ -261,6 +261,28 @@ def update_aggregates():
     logger.info('Finished updating incremental aggregates.')
 
 @manager.command
+def add_itemized_partition_cycle(cycle=None, amount=1):
+    """Adds a new itemized cycle child table.
+    By default this will try to add just the current cycle to all partitioned
+    itemized schedule tables if it doesn't already exist.
+    If the child table already exists, skip over it.
+    """
+
+    amount = int(amount)
```

Comment:
I was trying to find more info and while looking up more details I think it might actually be better to revisit these at some point, given the documentation here: https://flask-script.readthedocs.io/en/latest/#adding-arguments-to-commands

I was basing the arguments off of what we already had and I noticed they were coming in as strings, but by setting the options as outlined in the `flask-script` docs, there might be another way of doing this.  That said, this is also simple. :-)